app/scripts/modules/core/src/diffs/viewChangesLink.component.ts

```diff
@@ -7,7 +8,20 @@ import {COMMIT_HISTORY_COMPONENT} from './commitHistory.component';
 import {EXECUTION_SERVICE, ExecutionService} from 'core/delivery/service/execution.service';
 import {JAR_DIFF_COMPONENT, IJarDiff} from './jarDiff.component';
 
+export interface IJenkinsInfo {
+  host: string;
+  name: string;
+  number: string;
+}
+
+export interface IBuildInfo {
+  ancestor: string,
+  jenkins?: IJenkinsInfo;
+  target: string,
+}
```

Comment:
Feels like this is really a domain class associated with `IServerGroup` and could move in there, although the `ancestor` field is maybe specific to an entity tag on the server group.